crates/analytics/src/search.rs

```diff
@@ -26,23 +26,47 @@ pub async fn msearch_results(
         .execute(query_builder)
         .await
         .change_context(OpenSearchError::ConnectionError)?
-        .json::<OpenMsearchOutput<Value>>()
+        .json::<OpenMsearchOutput>()
```

Comment:
Can we read this as a string first using .text() and then try to parse it?
that way we can print the string as part of the parsing error helping us debug this in production environments

Reply:
Done